agent/lib/kontena/workers/container_health_check_worker.rb

```diff
@@ -9,7 +9,7 @@ class ContainerHealthCheckWorker
     include Kontena::Helpers::PortHelper
     include Kontena::Helpers::RpcHelper
 
-    HEALTHY_STATUSES = [200]
+    HEALTHY_STATUSES = (200 .. 208).to_a.freeze
```

Comment:
Should this just be `200...300` to also cover any custom codes up to 299?

The `to_a` should be unnecessary?

```
irb(main):005:0> (200...300).include? 206
=> true
```

Reply:
Should be `HEALTHY_STATUSES = (200...300)` to actually match any 2xx status codes, not just up to 208. There are [standardized](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success) HTTP response codes beyond 208.